go/ql/lib/semmle/go/Packages.qll

```diff
@@ -14,7 +14,7 @@ class Package extends @package {
   /** Gets the path of this package. */
   string getPath() {
     exists(string fullPath | packages(this, _, fullPath, _) |
-      result = fullPath.regexpReplaceAll("^.*/vendor/", "")
+      result = fullPath.regexpReplaceAll("^.*\\bvendor/", "")
```

Comment:
Just to check: it seems like this regex accepts anything preceding `vendor/` (with `.*`) so I am guessing the intention with the leading `/` was to filter out `bla/vendor/bar`, but not `foovendor/bar`. Should this be modified to account for that case again?

Reply:
I agree that's what we want. I believe the `\b` should only match boundaries, so I think it does do that. I think these links show that: https://www.debuggex.com/r/JDIU6t2gazjjM8MY, https://www.debuggex.com/r/86iT4pXqTA-IFCnq and https://www.debuggex.com/r/UHqBkd_Vlgm9eksk.

Reply:
I hadn't noticed the `\b` in there, sorry. That does look fine then.